README.md

```diff
@@ -4,22 +4,23 @@ This project is for sending slack/email/SQS messages in response to events. It s
 
 * ECR scan results. Each time there is an ECR repository scan, the scan results are checked. If there are any errors, a
   Slack and email message is sent.
-* Maintenance window results. When the Jenkins backup maintenance window runs, the results are checked and if it fails,
-  a notification is sent.
 * Consignment export results. When the consignment export task finishes, a Slack message is sent with details of whether
   the export succeeded or failed. A message is also sent, if it is a `judgment` export, to a SQS queue belonging to the transformation engine.
 * Transfer Engine retry event. If the Transform Engine requires a retry for any reason, it sends a message to the TDR Transform Engine SQS queue, which triggers the lambda to send a new export message back to a SQS queue belonging to the Transformation Engine.
   * Note: the Transfer Engine retry event does not tigger a re-export of the consignment.
+* Keycloak secret rotation messages. When the Keycloak secret rotation lambda runs, it will send success or failure messages to the notifications SNS topic.
 
 ## Run locally
 
 Set these environment variables, either on the command line or in IntelliJ depending on how you want to run the app:
 
 * `SLACK_WEBHOOK`: the webhook URL of a Slack app. You can [create a new app][Slack-app] (steps 1-3) in Slack for testing purposes.
-  Use the `#bot-testing` channel rather than a team channel to avoid confusion and spam.
+  Use the `#bot-testing` channel rather than a team channel to avoid confusion and spam. This webhook goes to the #da-tdr-notifications channel.
+* `SLACK_JUDGMENT_WEBHOOK` This webhook publishes to the #da-tdr-prod-exports-judgments channel.
```

Comment:
Could you just add an additional comment, that it should only be used to send production judgment transfer notifications?

Reply:
Ah good point. I've added it in.